rfcs/0002-ckb/0002-ckb.md

```diff
@@ -7,372 +7,234 @@ Organization: Nervos Foundation
+The `deps` and `inputs` in CKB transactions make it easier for nodes to determine transaction dependencies and perform parallel transaction processing (Figure 4). Different types of cells can be mixed and included in a single transaction to achieve atomic operation across types.
 
-Please check the CKB Consensus Paper for more details.
+## 5. Economic Model
 
-## 4. CKB Token
+A well-designed economic model should incentivize all participants to contribute to the success of the crypto-economy and maximize the utility of the blockchain.
 
-### 4.1  Economics
+The CKB economic model is designed to motivate users, developers and node operators to work towards the common goal of common knowledge custody. The subject of CKB economic model is state instead of computation, by using cell capacity and transaction fees as incentives for stakeholders.
 
-A well-designed economic model should incentivize all participants to contribute to the success of the community and maximize the utility of the blockchain.
+### 5.1 State Cost and Cell Capacity
 
-The Nervos Network is designed  to motivate users, developers and node operators to work towards the common goal of forming and storing common knowledge. The Protocol’s economic model focuses on states, by using increased Cell Capacity and transaction fees as incentives for stakeholders.
+The creation and storage of states on the CKB incur costs. The creation of new states needs to be verified by full nodes, incurring computational cost; the storage of states needs full nodes to provide storage space on an ongoing basis. Current permissionless blockchains only charge one-time transaction fees, but they allow states to be stored on all full nodes, occupying storage space forever.
 
-The creation and storage of states on the CKB incur cost. The creation of states needs to be validated by full nodes, incurring computational cost; the storage of states needs full nodes to provide storage space on an ongoing basis. Current permissionless blockchains only charge one time transaction fees, but they allow data to be stored on all full nodes, occupying storage space forever.
+In CKB, cells are basic storage units of states. Unoccupied cell capacity is transferable, and this gives cells liquidity. Users cannot transfer occupied cell capacity and lose liquidity on their occupied capacity. Therefore, cell owners pay for storage with the liquidity loss of cell capacity, which is an opportunity cost. The larger capacity and the longer time they occupy, the higher liquidity cost they pay. The advantage of pay by liquidity loss, compared to upfront payments, is that it avoids the problem that upfront payments could be used up, and the system would have to recycle the cells. Cells could have different owners and users, and owners can pay the liquidity cost on behalf of their users.
```

Comment:
> Users cannot transfer occupied cell capacity and lose liquidity on their occupied capacity.

This sentence makes me confused. The two clauses indeed are a cause-result relationship, so **thus** is a better choice than **and**.

Strictly speaking, users do be able to transfer occupied cell capacity. What users cannot is to transfer without losing the state.

Reply:
fixed.

Reply:
updated.